src/Admin/OptionsMenu.php

```diff
@@ -222,6 +223,14 @@ public function enqueue_assets( $hook_suffix ) {
 		$theme           = wp_get_theme();
 		$is_reader_theme = $this->reader_themes->theme_data_exists( get_stylesheet() );
 
+		$notices                      = get_option( PageCacheFlushNeededNotice::OPTION_NAME, [] );
+		$show_page_cache_flush_notice = ( in_array( PageCacheFlushNeededNotice::NOTICE_ID, $notices, true ) );
+
+		$site_has_cache_enable = (
+			( defined( 'WP_CACHE' ) && true === WP_CACHE ) ||
```

Comment:
Perhaps `has_page_caching()` should consider the `WP_CACHE` constant as a fallback?